package.json

```diff
@@ -155,7 +155,7 @@
     "test:integration:chrome": "node tests/integration/run.mjs --bchrome",
     "test:integration:chrome:watch": "node tests/integration/run.mjs --bchrome --watch",
     "test:integration:coverage": "node tests/integration/run.mjs --bchromehl --bfirefoxhl --coverage",
-    "test:integration:firefox": "node tests/integration/run.mjs --bfirefox",
+    "test:integration:firefox": "node tests/integration/run.mjs --bfirefoxhl",
```

Comment:
Both the chrome and firefox variations here were I think intended to rely on the non-headless version to simplify debugging (you can easily open a debugger on that browser).